src/llamacrawl/services/readers/sessions/manager.py

```
@@ -156,15 +157,15 @@ def start(self) -> None:
                         "machine": machine_config.name,
                         "paths": watch_paths,
                         "interval": machine_config.poll_interval_seconds,
-                    }
+                    },
                 )
 
             except Exception as e:
                 # Log error but don't fail - allow other remote watchers to start
                 self.logger.error(
```

Ruff (0.14.0)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)